scripts/package

```diff
@@ -60,6 +60,33 @@ initrd \${url}/\${initrd}
 boot
 EOF
 
+GITHUB_REPOSITORY="${GITHUB_REPOSITORY:-rancher-sandbox/os2}"
+UPGRADECHANNEL_DISCOVERY_IMAGE="${UPGRADECHANNEL_DISCOVERY_IMAGE:-quay.io/costoolkit/upgradechannel-discovery:v0.3-4b83dbe}"
+for i in "amd64"; do
+cat > build/output.upgradechannel-$i.yaml << EOF
+apiVersion: rancheros.cattle.io/v1
+kind: ManagedOSVersionChannel
+metadata:
+  name: os2-$i
+  namespace: fleet-default
+spec:
+  options:
+    args:
+    - github
```

Comment:
I opted out for scanning just github releases here with rancher-sandbox/os2. Both for sake of semplicity on our side and flexibility on user side. As this is for the community one, we should be good with it to keep synced to always latest available versions.

For other more convoluted scenarios (which might involve obs, et all) we can use the `git` syncer instead so we are in control of what versions are going to be pushed